modules/ROOT/pages/depl-examples/bare-metal.adoc

```diff
@@ -73,6 +73,21 @@ Note that this URL must resolve to the server running this installation.
 
 === Install and configure the Infinite Scale Binary
 
+[NOTE]
+====
+If you have installed Infinite Scale already but want to upgrade it, do the following steps first:
+
+* xref:stopping-infinite-scale[Stop infinite Scale]: +
+You cannot overwrite the binary of a running instance.
+
+* Make a copy of the existing binary first:
```

Comment:
Making a copy in this way does not help. the original binary file is still locked for swapping by the kernel.

I'd prefer to move away the running binary - then we can add a new version with the original name.
E.g. the following commands can be done without stopping ocis:
```
  wget -P /usr/local/bin https://download.owncloud.com/ocis/ocis/testing/7.0.0-rc.1/ocis-7.0.0-rc.1-linux-amd64
  chmod a+x /usr/local/bin/ocis-7.0.0-rc.1-linux-amd64
  mv /usr/local/bin/ocis /usr/local/bin/ocis-5.0.8      # it can happily continue to run under this name.
  ln -s ocis-7.0.0-rc.1-linux-amd64 /usr/local/bin/ocis

  ocis version --skip-services
    Version: 7.0.0-rc.1
    Compiled: 2024-11-07 00:00:00 +0000 UTC
```

The symlink is used here to document the versions, active and inactive.
(A simpler sequence - with a longer absence - would just use wget -O after mv.)

Reply:
This procedure is a good idea, if that works - needs testing, I like it.
With that, we can easily switch versions and keep track of original naming.

Reply:
Works fine for me :man_shrugging: at least. 
Symlinks sometimes cause confusion; but here it looks smooth, and I could not think of anything that would break.

Reply:
Fixed via #1055 (Update bare metal deployments)